bot/cogs/core/database.py

```diff
@@ -42,6 +42,9 @@ async def populate_caches(self):
     async def fetch_user_reminder_count(self, user_id: int) -> int:
         return await self.db.fetchval("SELECT COUNT(*) FROM reminders WHERE user_id = $1", user_id)
 
+    async def fetch_user_reminders(self, user_id: int):
```

Comment:
Generally we typehint the return of a function here

Reply:
I should type hint it to a list right?